schema.graphql

```diff
@@ -44,6 +44,11 @@ enum JobStatus {
   FAILED
 }
 
+type fdrCountsNumber {
```

Comment:
GraphQL types must start with a capital letter
see examples http://graphql.org/learn/schema/#object-types-and-fields

Reply:
@intsco thanks for pointing on that problem and links. I will correct it